gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/FormatHelper.java

```diff
@@ -92,7 +93,8 @@ public static String formatGrade(final String grade) {
 	 * @return
 	 */
 	private static String formatDate(final Date date) {
-		final SimpleDateFormat df = new SimpleDateFormat("MM/dd/yyyy"); // TODO needs to come from i18n
+		final String dateFormatString = new ResourceModel("format.date").getObject();
```

Comment:
I think this may fail under certain circumstances as `ResourceModel` is Wicket only, and if this method was ever called outside of that context it would die. We have `MessageHelper.getString` that does this outside of the Wicket context, maybe switch it over?